src/store/annotations/__tests__/reducer-test.ts

```diff
@@ -36,4 +41,22 @@ describe('store/annotations/reducer', () => {
             expect(newState.activeId).toBe(payload);
         });
     });
+
+    describe('removeAnnotatorAction', () => {
+        test('should delete an annotation from the store', () => {
+            const payload = 'test1';
+            const newState = reducer(state, removeAnnotationAction(payload));
+
+            expect(newState.allIds).not.toContain(payload);
+            expect(newState.byId.test1).toBe(undefined);
+        });
+
+        test('should set activeId to null if deleted annotation is active', () => {
+            const payload = 'test1';
+            const stateWithActiveId = reducer(state, setActiveAnnotationIdAction(payload));
```

Comment:
Instead of bringing `setActiveAnnotationIdAction` into this test, could we instead just extend `state` to include an `activeId`?